### PR TITLE
Added a '--check' flag to 'jag setup' to validate setup

### PR DESCRIPTION
**Pull request by jstroem**
Fixes: #79

### CODE REVIEW

File: cmd/jag/commands/setup.go [hunk 1]
```diff
@@ -54,6 +55,31 @@ func SetupCmd(info Info) *cobra.Command {
 		SilenceUsage: true,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			ctx := cmd.Context()
+			check, err := cmd.Flags().GetBool("check")
+			if err != nil {
+				return err
+			}
+
+			if check {
```
Comment (kasperl) [inline cmd/jag/commands/setup.go:63]: Will returning with an error from there give an exit code of 1?

**Review by kasperl — APPROVED** [review]
Looks good to me!

File: cmd/jag/commands/setup.go (hunk 1 above)
Comment (jstroem) [reply cmd/jag/commands/setup.go:63]: yes.

Comment (jstroem) [reply cmd/jag/commands/setup.go:63]: not it will :)

**Pull request merged**